dart/pubspec.yaml

```diff
@@ -0,0 +1,13 @@
+name: gurmukhi_utils
+description: Utilities for Gurmukhi text.
+version: 1.0.0
```

Comment:
```suggestion
version: 0.0.1
```

Reply:
I think its better to use 0.1.0, haven't seen any dart package with 0.0.x version.